x-pack/plugins/ingest_manager/server/services/epm/registry/registry_url.ts

```diff
@@ -3,26 +3,43 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { DEFAULT_REGISTRY_URL } from '../../../constants';
 import { appContextService, licenseService } from '../../';
 
+// from https://github.com/elastic/package-registry#docker (maybe from OpenAPI one day)
+// the unused variables cause a TS warning about unused values
+// chose to comment them out vs @ts-ignore or @ts-expect-error on each line
+
+const PRODUCTION_REGISTRY_URL_CDN = 'https://epr.elastic.co';
+// const STAGING_REGISTRY_URL_CDN = 'https://epr-staging.elastic.co';
+// const EXPERIMENTAL_REGISTRY_URL_CDN = 'https://epr-experimental.elastic.co/';
```

Comment:
Experimental should not be here as this will never apply for master or anything coming out of master. Same on line 19.